web-components/src/components/usa-banner/index.js

```diff
@@ -20,19 +31,146 @@ export class UsaBanner extends LitElement {
+        },
+        domain: {
+          heading: "Official websites use",
+          text1: "A",
+          text2:
+            "website belongs to an official government organization in the United States.",
+        },
+        https: {
+          heading1: "Secure",
+          heading2: "websites use HTTPS",
+          text1: "A <strong>lock</strong>",
+          text2:
+            "or <strong>https://</strong> means you’ve safely connected to the",
+          text3:
+            "website. Share sensitive information only on official, secure websites.",
+        },
+      },
+      es: {
+        banner: {
+          label: "Un sitio oficial del Gobierno de Estados Unidos",
+          text: "Un sitio oficial del Gobierno de Estados Unidos",
+          action: "Así es como usted puede verificarlo",
+        },
+        domain: {
+          heading: "Los sitios web oficiales usan",
+          text1: "Un sitio web",
+          text2:
+            "pertenece a una organización oficial del Gobierno de Estados Unidos.",
+        },
+        https: {
+          heading1: "Los sitios web seguros",
+          heading2: "usan HTTPS",
+          text1: "Un <strong>candado</strong>",
+          text2:
+            "o <strong>https://</strong> significa que usted se conectó de forma segura a un sitio web",
+          text3:
+            "Comparta información sensible sólo en sitios web oficiales y seguros.",
+        },
+      },
+    };
+  }
+
+  // Get English or Spanish strings. Default to English if an unknown `lang` is passed.
+  // Ex: <usa-banner lang="zy"></usa-banner>
+  get _bannerText() {
+    const content = this.data[this.lang] || this.data["en"];
+    return content;
   }
 
-  static styles = [unsafeCSS(usaBannerStyle)];
+  // Get the action text and use for both mobile & desktop buttons.
+  get _actionText() {
+    const bannerActionText = this.querySelector('[slot="banner-action"]');
+
+    return bannerActionText?.textContent;
+  }
+
+  // TODO: Use inline image instead or translate strings.
+  svgLock() {
+    return html`
+      <span class="icon-lock">
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          width="52"
+          height="64"
+          viewBox="0 0 52 64"
+          class="usa-banner__lock-image"
+          role="img"
+          aria-labelledby="banner-lock-description-default"
+          focusable="false"
+        >
+          <title id="banner-lock-title-default">Lock</title>
+          <desc id="banner-lock-description-default">Locked padlock icon</desc>
+          <path
+            fill="#000000"
+            fill-rule="evenodd"
+            d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z"
+          />
+        </svg>
+      </span>
+    `;
+  }
+
+
+  // ! CSS won't work if comments added inside css``.
```

Comment:
If we're going to standardize on keeping styles styles, the (S)CSS should probably go in a separate file. For `usa-link` I just moved the styles into a `usa-link.css.js`, but maybe here it might make sense to do `usa-banner.scss.js` to signal that there's still a SASS dependency here.

Reply:
Separated styles in 3549483.

> [!CAUTION]
> My only concern here is that a developer could see this and write SCSS instead of CSS, which is what we're moving towards.